### PR TITLE
// Update faceted search with navigation element. REF PR: #4834

### PR DESCRIPTION
**Pull request by inetbiz**
Read: http://www.w3schools.com/tags/tag_nav.asp

### CODE REVIEW

File: themes/StarterTheme/templates/catalog/_partials/facets.tpl [hunk 1]
```diff
@@ -70,6 +71,7 @@
             {/if}
           </form>
         {/if}
+        </nav>
```
Comment (maximebiloe) [inline themes/StarterTheme/templates/catalog/_partials/facets.tpl]: Can you move your `</nav>` tag inside the `{if}`please ?

**Comment by xBorderie** [comment]
Even better: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav ;)

**Comment by inetbiz** [comment]
@xBorderie so you agree with me? I understand now that these display the feature or option label in a section of faceted search. This will fulfill Google's patent related to Entity relatedness in close proximity and signal the content type is navigation for screen readers.

**Comment by xBorderie** [comment]
Rather than agreeing with you, it was more of a question of pointing to MDN rather than to w3school, which I hate :) [See here](http://www.w3fools.com/).
Sorry! The whole decision-making about this PR will be done by the developers -- of which I am not :)

**Comment by inetbiz** [comment]
@xBorderie :+1:

**Comment by maximebiloe** [comment]
Actually, you need to close the `<nav>` element in the same section of the `if`. So you need to do it before the `else`. Sorry, I was not clear before.

**Comment by xBorderie** [comment]
Thank you for this PR, @inetbiz, but as you can see it was not made in a up-to-date repo.
Please redo the PR so that it only contains your 29c4d195fcd754909051809b3c1dd5f0daadb85a commit.

**Pull request closed without merging**